pkgs/tools/networking/subfinder/default.nix

```diff
@@ -22,8 +22,8 @@ buildGoPackage {
       subdomains for websites. Designed as a passive framework to be
       useful for bug bounties and safe for penetration testing.
     '';
-    homepage = https://github.com/subfinder/subfinder;
+    homepage = "https://github.com/projectdiscovery/subfinder";
```

Comment:
This could also be taken from `src.meta.homepage`.

Reply:
I believe this is [taken by default](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchgithub/default.nix#L33). Should this be spelled out explicitly?

Reply:
Perhaps not, I did not know it was taken by `mkDerivation`.

Reply:
I think that it serves well for documentation purposes, and is similar to most other derivations.